shared/middleware/helpers/auth_helpers.rb

```diff
@@ -18,6 +18,14 @@ def current_user
   @dashboard_user ||= DASHBOARD_DB[:users][id: current_user_id]
 end
 
+# Returns true if the current user is under 13 or if age is unknown.
+def under_13?
+  return true unless current_user
+  birthday = current_user[:birthday]
+  age = ((Date.today - birthday) / 365).to_i
+  age < 13
+end
```

Comment:
This seems to be duplicating [`User#under_13?`](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/user.rb#L769-L771) in a sense. Is it possible to share the code? If not, I'd recommend adding a comment in both places indicating that the code is duplicated.